tools/build_pak_files.py

```diff
@@ -0,0 +1,129 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+import argparse
+import glob
+import os
+import posixpath
+import spear
+import subprocess
+import sys
+
+
+if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--contents_dir", required=True)
+    parser.add_argument("--unreal_binaries_dir", required=True)
+    parser.add_argument("--platforms", nargs="*", required=True)
+    parser.add_argument("--output_dir", required=True)
+    parser.add_argument("--unreal_project_dir")
+    args = parser.parse_args()
+    
+    assert os.path.exists(args.unreal_binaries_dir)
+    if sys.platform == "win32":
+        unreal_editor_bin = os.path.realpath(os.path.join(args.unreal_binaries_dir, "Win64", "UE4Editor.exe"))
+        unreal_pak_bin    = os.path.realpath(os.path.join(args.unreal_binaries_dir, "Win64", "UnrealPak.exe"))
+    elif sys.platform == "darwin":
+        unreal_editor_bin = os.path.realpath(os.path.join(args.unreal_binaries_dir, "Mac", "UE4Editor"))
+        unreal_pak_bin    = os.path.realpath(os.path.join(args.unreal_binaries_dir, "Mac", "UnrealPak"))
+    elif sys.platform == "linux":
+        unreal_editor_bin = os.path.realpath(os.path.join(args.unreal_binaries_dir, "Linux", "UE4Editor.sh"))
+        unreal_pak_bin    = os.path.realpath(os.path.join(args.unreal_binaries_dir, "Linux", "UnrealPak.sh"))
+    
+    if args.unreal_project_dir:
+        unreal_project_dirs = [os.path.realpath(args.unreal_project_dir)]
+    else:
+        unreal_projects_dir = os.path.realpath(os.path.join(os.path.dirname(os.path.realpath(__file__)), "..", "cpp", "unreal_projects"))
+        unreal_project_dirs = [ os.path.join(unreal_projects_dir, project) for project in os.listdir(unreal_projects_dir) ]
+    
+    assert os.path.exists(args.contents_dir)
+    content_dirs = [os.path.realpath(os.path.join(args.contents_dir, x)) for x in os.listdir(args.contents_dir)]
```

Comment:
Pad with whitespace for list comprehensions.

```
[ os.path.realpath(os.path.join(args.contents_dir, x)) for x in os.listdir(args.contents_dir) ]
```

Reply:
Done